### ISSUE TITLE
include_cache doesn't update when using Auto-regeneration

### ISSUE THREAD

**Issue by fxedel** [benbalter/jekyll-include-cache#1]
**Steps to reproduce:**

1. Create a file `_includes/my-include.html`:
````
<h3>Super Heading!</h3>
````
2. Create a file `my-page.md`:
````
---
title: My Page
permalink: /my-page/
---

{% include_cached my-include.html %}
````
3. Run `jekyll serve`
4. Change `_includes/my-include.html` to:
````
<h3>Another Heading!</h3>
````
5. :zap: The actual website doesn't update and still shows "Super Heading!"

**Expected/Wished behavior:**

5. The website *does* update and shows "Another Heading!"

**Comment by thelegy** [comment]
Hey, would you mind updating the gem to also incorporate the changes there?

**Comment by WORMSS** [comment]
My co-worker is getting this caching problem during development.
I see you have made a fix for this in RP #2 .
We are not sure if this is a problem that he has a `include_cached` inside a file that has `include_cached` and he is modifying the nested file?

Sadly I know nothing of jekyll and someone else set this up for him. He just modifies files until the website looks correct.
But he has to keep closing the serve and then run serve again to get this to work.
